src/components/Timeline/Timeline.stories.tsx

```diff
@@ -5,23 +5,50 @@ import { ComponentMeta, ComponentStory } from '@storybook/react';
 
 const chance = Chance();
 
+import mockNote from 'utils/testing/mocks/mockNote';
 import mockUpdate from 'utils/testing/mocks/mockUpdate';
-import Timeline from 'components/Timeline';
-import { UPDATE_TYPES, ZetkinUpdateJourneyMilestone } from 'types/updates';
+import Timeline from 'components/Timeline/index';
+import {
+  UPDATE_TYPES,
+  ZetkinUpdateJourneyInstance,
+  ZetkinUpdateJourneyInstanceAddNote,
+  ZetkinUpdateJourneyInstanceMilestone,
+} from 'types/updates';
 
 export default {
   argTypes: {
     backgroundColor: { control: 'color' },
   },
   component: Timeline,
-  title: 'Example/Timeline',
+  title: 'Organisms/Timeline',
 } as ComponentMeta<typeof Timeline>;
 
 // More on component templates: https://storybook.js.org/docs/react/writing-stories/introduction#using-args
 const Template: ComponentStory<typeof Timeline> = (args) => (
-  <Timeline {...args} showAll />
+  <div style={{ maxWidth: 450 }}>
+    <Timeline {...args} showAll />
+  </div>
 );
 
+const journeyInstanceUpdates = Array.from(Array(10).keys()).map(() => {
```

Comment:
I found it difficult when developing in Storybook to use this type of random data, because it meant that with every reload the timeline looked different. So I just added a single static update to the mock data when I implemented the `journeyinstance.create` event.

I would suggest at some point just creating a single const `updates` array with one of each variation, instead of randomizing, concatenating, and sorting.

Reply:
Leaving this for now because it seems low priority :)